Essentials/src/main/resources/config.yml

```diff
@@ -904,39 +895,43 @@ sell-multipliers:
 # +------------------------------------------------------+ #
 ############################################################
 
-# Show other plugins commands in help.
+# Show other plugins' commands in the Essentials help list.
 non-ess-in-help: true
 
-# Hide plugins which do not give a permission.
-# You can override a true value here for a single plugin by adding a permission to a user/group.
-# The individual permission is: essentials.help.<plugin>, anyone with essentials.* or '*' will see all help regardless.
-# You can use negative permissions to remove access to just a single plugins help if the following is enabled.
+# Hide plugins that players do not have permission to use.
+# You can override this by adding the 'essentials.help.<plugin>' permission to a player or group.
+# Players with 'essentials.*' or '*' will see all help regardless.
+# You can also use negative permissions to remove access to a specific plugin's help if this is enabled.
 hide-permissionless-help: true
 
 ############################################################
 # +------------------------------------------------------+ #
-# |                   EssentialsX Chat                   | #
+# |                   Essentials Chat                    | #
 # +------------------------------------------------------+ #
 ############################################################
 
-# You need to install EssentialsX Chat for this section to work.
+# You need to install the EssentialsX Chat module for this section to work.
 # See https://essentialsx.net/wiki/Module-Breakdown.html for more information.
 
 chat:
 
-  # If EssentialsX Chat is installed, this will define how far a player's voice travels, in blocks. Set to 0 to make all chat global.
-  # Note that users with the "essentials.chat.spy" permission will hear everything, regardless of this setting.
-  # Users with essentials.chat.shout can override this by prefixing their message with an exclamation mark (!)
-  # Users with essentials.chat.question can override this by prefixing their message with a question mark (?)
-  # You can add command costs for shout/question by adding chat-shout and chat-question to the command costs section.
+  # If Essentials Chat is installed, this sets how many blocks a player's chat will travel. Set to 0 for global chat.
```

Comment:
I'd prefer to keep the EssentialsX name in any references to module names, to prevent someone looking for module jars from upstream.

(I'm a little mixed on removing the EssentialsX branding in general, but that's probably a separate discussion)